composer.json

```diff
@@ -7,7 +7,8 @@
     "require": {
         "php": "^7.1.3",
         "fideloper/proxy": "^4.0",
-        "laravel/framework": "5.8.*",
+        "laravel/framework": "6.*",
+        "laravel/helpers": "^1.1",
```

Comment:
Just want to ask, do we need this `laravel/helpers` package? Are there any error happen if we don't include that @AnBusse?